src/Http/Handlers/AuthorizeHandler.php

```diff
@@ -36,6 +36,7 @@ public function handle( Request $request, Response $response ): Response {
 
 		$user = wp_get_current_user();
 		if ( $this->consent_storage->needs_consent( $user->ID ) ) {
+			// phpcs:disable WordPress.Security.NonceVerification.Recommended We already have REST API nonce validating the user intention with this request
```

Comment:
I think `phpcs:disable` applies to the whole file, you could use `phpcs:ignore` to ignore the next line.